tools/lint/cmd_print.c

```diff
@@ -185,6 +190,12 @@ cmd_print(struct ly_ctx **ctx, const char *cmdline)
             }
             break;
 
+        case 'L': /* --tree-line-length */
+            if (sscanf(optarg, "%zu", &line_length) != 1) {
```

Comment:
Avoid `scanf` and any associated functions, they are often doing what they want and not what you expect. Instead, you can use `atoi()` when you do not care about an error that much or `strtoul()` when you do.